client/state/rewind/size/test/reducer.js

```diff
@@ -61,5 +65,23 @@ describe( 'rewind.size reducers', () => {
 				expect( sizeReducer( state, action ).retentionDays ).toEqual( expected );
 			}
 		);
+
+		it.each( [
+			{
+				state: undefined,
+				action: { type: JETPACK_BACKUP_RETENTION_SET, retentionDays: 30 },
+				expected: 30,
+			},
+			{
+				state: { retentionDays: 30 },
+				action: { type: JETPACK_BACKUP_RETENTION_SET, retentionDays: 7 },
+				expected: 7,
+			},
+		] )(
+			'should return the new retentionDays value if we set it using the action JETPACK_BACKUP_RETENTION_SET',
```

Comment:
Note that it's useful to have the different values inside the test's name if we're doing this, otherwise when this test fails it will display a cryptic message. 

That's why I generally recommend avoiding `it.each()` unless it really saves a lot of effort. Here it only does a simple `expect()` assertion so we're not saving much code. That's the other thing about testing: saving code often means we sacrifice readability and comprehensibility.

Reply:
Yeah, I think we can get rid of it.each in this scenario. Just made the change :)
Thank you for the suggestion!